docs/site/Rest-Crud-generator.md

```diff
@@ -0,0 +1,76 @@
+---
+lang: en
+title: 'REST CRUD model endpoints generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Rest-Crud-generator.html
+---
+
+{% include content/generator-create-app.html lang=page.lang %}
+
+### Synopsis
+
+Adds a new
+[Rest config for model endpoint (or multiple backed by the same datasource)](Creating-CRUD-REST-apis.md)
+to a LoopBack application with one single command.
+
+```sh
+lb4 rest-crud [options]
+```
+
+### Options
+
+`--datasource` : _(Optional)_ name of a valid datasource already created in
+src/datasources
+
+`--model` : _(Optional)_ name of a valid model already created in src/models
+
+### Configuration file
+
+This generator supports a config file with the following format, see the
+Standard options below to see different ways you can supply this configuration
+file.
+
+```ts
+{
+  "datasource": "validDataSourceName",
+  "model": "validDModelName"
+}
+```
+
+### Notes
+
+Service oriented datasources such as REST or SOAP are not considered valid in
+this context and will not be presented to you in the selection list.
+
+There should be at least one valid _(KeyValue or Persisted)_ data source and one
+model already created in their respective directories.
+
+{% include_relative includes/CLI-std-options.md %}
+
+### Interactive Prompts
+
+The tool will prompt you for:
+
+- **Please select the datasource.** _(name)_ If the name of the datasource had
+  been supplied from the command line, the prompt is skipped, otherwise it will
+  present you the list of available datasources to select one. It will use this
+  datasource to check what kind of repository it will generate.
+
+- **Select the model(s) you want to generate a repository.** _(model)_ If the
```

Comment:
I think this was "Select the model(s) you want to generate a CRUD REST endpoint" before if I'm not mistaken. Why was it changed to generating a repository?